contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -42,6 +42,19 @@ Distro-specific one-liners
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. tabs::
+    .. tab:: All distros x84_64
```

Comment:
This is worth documenting, but this is a bit too power user with only a script and no explanations about what this is for and why it's a good option.

It's also not a distro one-liner.

I'd make a separate section for it, and explain the purpose of this Linux SDK and why we recommend it for production builds.

Reply:
Made a new section at the very end of the documentation